match_with_noaa_stations.py

```diff
@@ -8,26 +36,41 @@
 import pandas as pd
 import requests
 
+""" Set the Path of the Input Files """
+NOAA_STATIONS_PATH = "station_inventory_cheater.csv"
+CBC_FILE_PATH = "cleaned_cbc_usa.csv"
+
+"""## Load in the NOAA Station Data"""
+
+noaa_sations = pd.read_csv(NOAA_STATIONS_PATH, encoding = "latin-1")
```

Comment:
These should not be in the global environment. Please adjust the appropriate function.